app/src/androidTest/java/com/github/polypoly/app/map/LocationRepositoryTest.kt

```diff
@@ -1,17 +1,17 @@
 package com.github.polypoly.app.map
 
+import com.github.polypoly.app.game.Zone
 import org.junit.Test
 
 // These tests are not really useful, but they're here symbolically, this is only a data class
-class LocalizationRepositoryTest {
+class LocationRepositoryTest {
     @Test
     fun testGetLocalizationsWorks() {
-        assert(LocalizationRepository.getZones().flatMap { zone -> zone.localizations }
-            .isNotEmpty())
+        assert(LocationRepository.getZones().flatMap(Zone::locations).isNotEmpty())
```

Comment:
excellent example of cleaning up code while you're at it :)

Reply:
It's still the wrong assert function though